hockeysdk/src/main/java/net/hockeyapp/android/UpdateManagerListener.java

```diff
@@ -16,16 +16,6 @@
  **/
 @SuppressWarnings({"WeakerAccess"})
 public abstract class UpdateManagerListener {
-    /**
-     * Return your own subclass of UpdateActivity for customization.
-     *
-     * @return subclass of UpdateActivity
-     */
-    @Deprecated
-    @SuppressWarnings("unused")
-    public Class<? extends UpdateActivity> getUpdateActivityClass() {
```

Comment:
It deprecated only since 5.0
Maybe we should wait a bit?

Reply:
you are right! Let me update that.